### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by cstewatest**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,119 +1,167 @@
...
+  onSelectOptions = selectedIDs => {
+    store.dispatch({
+      type: actionTypes.OPTIONS_SUBMITTED,
+      payload: { selected: selectedIDs }
+    });
     // make this the same requester throughout the code. no need to reinstantiate
-    let requester = new Requester(this.state.gender, this.state.birthYear) 
-    requester.get()
-  }
+    let requester = new Requester(this.state.gender, this.state.birthYear);
```
Comment (shellyyip) [inline App.js:104]: I read a thing about the singleton pattern! https://k94n.com/es6-modules-single-instance-pattern

File: src/components/MedicalInfo.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Breadcrumbs from './Breadcrumbs'
+
+const MedicalInfo = (props) => {
+  if (props.isFething || !props.hasSubmittedMedicalInfo) {
```
Comment (shellyyip) [inline src/components/MedicalInfo.js]: Oh no typo! `isFething`

File: src/constants/FormStyles.js [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
   },
-  button: { 
-    marginTop: 12, 
+  button: {
+    marginTop: 12,
     height: 40,
-    backgroundColor: '#ffffff',
-    borderColor: '#ffffff',
+    backgroundColor: "#ffffff",
+    borderColor: "#ffffff",
```
Comment (shellyyip) [inline src/constants/FormStyles.js:21]: Maybe a lil nit picky, but I noticed in some other places you used the named color (`white`) and here you used the hex; I’d just stick with one or the other as your convention.

File: src/reducers/unjaniRedux.js [hunk 4]
```diff
@@ -1,134 +1,149 @@
...
 
 export const prompts = {
-  BODY_LOCATION: 'Which part of your body hurts?',
-  BODY_SUBLOCATION: 'Specifially, which of these body parts hurts?',
-  SUBLOCATION_SYMPTOMS: 'Which of these symptoms are you having?',
-  ADDITIONAL_SYMPTOMS: 'Are you having any additional symptoms?',
-  DIAGNOSES: 'POTENTIAL DIAGNOSES'
-}
+  BODY_LOCATION: "Which part of your body hurts?",
+  BODY_SUBLOCATION: "Specifially, which of these body parts hurts?",
```
Comment (shellyyip) [inline src/reducers/unjaniRedux.js]: Oops typo! `Specifially`

Comment (shellyyip) [reply src/reducers/unjaniRedux.js]: Also grammar error: should be `which of these body parts hurt`

File: src/services/Requester.js [hunk 5]
```diff
@@ -1,89 +1,133 @@
...
+  store,
+  actionTypes,
+  stages,
+  noneOption
+} from "../reducers/unjaniRedux";
+import moment from "moment";
 
 export default class Requester {
-  BASE_URL = "http://ec2-54-204-218-134.compute-1.amazonaws.com/"
+  BASE_URL = "http://ec2-54-227-226-211.compute-1.amazonaws.com/"
```
Comment (shellyyip) [inline src/services/Requester.js]: Hmm does this URL need to be secured?

**Review by shellyyip — APPROVED** [review]
YAY this looks awesome, thanks for incorporating all my feedback! I just had a few non-critical questions and saw a few typos. I think one other thing to think about is what to do in the `Requester` helper if the request fails, other than just console logging the error out.

File: src/services/Requester.js (hunk 5 above)
Comment (cstewatest) [reply src/services/Requester.js]: Unfortunately you have to pay to secure your public URL on AWS if you want it to remain the same when you start/stop the box. I've been doing that to save $, but once I deploy it I'm just going to leave it on. Good catch though!

File: src/reducers/unjaniRedux.js (hunk 4 above)
Comment (cstewatest) [reply src/reducers/unjaniRedux.js]: Thank you!

**Pull request merged**